.gitignore

```diff
@@ -2,4 +2,6 @@ node_modules/
 bower_components/
 config/config.js
 rise-financial-version.js
+rise-financial.html
```

Comment:
@donnapep Not sure I follow how a designer running `gulp build` will resolve it. Can you clarify? I see this as a different situation to the `dist` folder. When a designer has created a project that uses this component, if we .gitignore these files, then these files will not be in the bower (or similar) package for their project.